en/docs/authentication-and-authorization/secure-api-access-with-azure-ad.md

```diff
@@ -17,16 +17,16 @@ This guide walks you through the following steps:
 Before you try out this guide, complete the following:
```

Comment:
Should we rephrase this sentence so that it aligns better with the given prerequisites? Something along the lines of 
"To try out this guide, you must have the following prerequisites in place:"

Reply:
reworded to To follow this guide, you need to satisfy the following prerequisites: